dev/ci/nix/default.nix

```diff
@@ -104,7 +104,6 @@ let projects = {
   fiat_crypto = callPackage ./fiat_crypto.nix {};
   flocq = callPackage ./flocq.nix {};
   formal-topology = callPackage ./formal-topology.nix {};
-  GeoCoq = callPackage ./GeoCoq.nix {};
```

Comment:
There is still a `GeoCoq.nix` file in the folder (although I honestly do not care if the cleanup goes as far as the `dev/ci/nix` folder, which is entirely untested anyway).